source/native/symbulation.cc

```diff
@@ -79,33 +79,36 @@ int symbulation_main(int argc, char * argv[])
 
   //This parameter is redundant with HOST_REPRO_RES, SYM_HORIZ_TRANS_RES, and SYM_LYSIS_RES.
```

Comment:
We can just remove these two lines of comments while you're at it, they aren't helpful